MekHQ/data/universe/defaultspa.xml

```diff
@@ -78,11 +78,23 @@
 		<lookupName>melee_master</lookupName>
 		<xpCost>15</xpCost>
 		<weight>6</weight>
-		<skillPrereq>
+        <prereqAbilities>melee_specialist</prereqAbilities>
+        <invalidAbilities>zweihander</invalidAbilities>
+        <skillPrereq>
 	    	<skill>Piloting/Mech::Regular</skill>
 	        <skill>Gunnery/Protomech::Regular</skill>
 	    </skillPrereq>
 	</ability>
+    <ability>
+        <lookupName>zweihander</lookupName>
+        <xpCost>10</xpCost>
+        <weight>6</weight>
+        <prereqAbilities>melee_specialist</prereqAbilities>
+        <invalidAbilities>melee_master</invalidAbilities>
```

Comment:
Its not exactly clear from Camo Ops, but it seems like the intent is that Zweihander and melee master are intended to be mutually exclusive. Users can change this if they wish.